react/features/whiteboard/actions.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable arrow-body-style */
+import {
+    HIDE_WHITEBOARD,
+    OPEN_WHITEBOARD,
+    RESET_WHITEBOARD,
+    SETUP_WHITEBOARD,
+    SHOW_WHITEBOARD
+} from './actionTypes';
+
+/**
+ * Configures the whiteboard collaboration details.
+ *
+ * @param {Object} payload - The whiteboard settings.
+ * @returns {{
+ *     type: SETUP_WHITEBOARD,
+ *     collabDetails: { roomId: string, roomKey: string }
+ * }}
+ */
+export const setupWhiteboard = ({ collabDetails }: {
+    collabDetails: { roomId: string; roomKey: string; };
+}): any => ({
+    type: SETUP_WHITEBOARD,
+    collabDetails
+});
+
+/**
+ * Starts the whiteboard collaboration.
+ *
+ * @returns {{
+ *     type: OPEN_WHITEBOARD
+ * }}
+ */
+export const openWhiteboard = (): any => ({ type: OPEN_WHITEBOARD });
```

Comment:
There are way too many actions here :-) There is open, then show then hide. A single SET_WHITEBOARD_VISIBLE should suffice, right? It would take a `visible` boolean.

Reply:
Agree, I removed show and hide for now, I'll deal with open when switching from presence to metadata.